README.md

```diff
@@ -169,12 +176,15 @@ An example of extra specs json would look like this:
     "nic_type": "VIRTIO_NET",
     "custom_labels": {"environment":"production","project":"myproject"},
     "network_tags": ["web-server", "production"],
-    "source_snapshot": "projects/garm-testing/global/snapshots/garm-snapshot"
+    "source_snapshot": "projects/garm-testing/global/snapshots/garm-snapshot",
+    "ssh_keys": ["username1:ssh_key1", "username2:ssh_key2"]
 }
 ```
 
 **NOTE**: The `custom_labels` and `network_tags` must meet the [GCP requirements for labels](https://cloud.google.com/compute/docs/labeling-resources#requirements) and the [GCP requirements for network tags](https://cloud.google.com/vpc/docs/add-remove-network-tags#restrictions)!
 
+**NOTE**: The `ssh_keys` add the option to [connect to an instance via SSH](https://cloud.google.com/compute/docs/instances/ssh) (either Linux or Windows). After you added the key as `username:ssh_public_key`, you can use the `private_key` to connect to the Linux/Windows instance via `ssh -i private_rsa username@instance_ip`.
```

Comment:
Do the images need any specific packages pre-installed? Is there a quick howto you can include in the README?

Reply:
I considered that if you want to use ssh keys on Windows, you want to have ssh enabled and installed on the instance

Reply:
it's worth adding a note that this also works on Windows, if the needed packages are pre-installed in the image.

Reply:
Mentioned at the end of the `NOTE` that the provider installs on the instance `google-compute-engine-ssh` and `enables ssh` if a ssh key is sent as extra-spec